docs/UserGuide.adoc

```diff
@@ -649,120 +689,84 @@ Examples:
-e.g. `orderadd 1 i/NBA 2k18 pr/229.99 q/1 d/11-09-2018`
-* *OrderEdit* : `orderedit INDEX [i/ORDER INFORMATION] [pr/PRICE] [q/QUANTITY] [d/DELIVERY DATE]` +
-e.g. `orderedit 1 pr/15.00 d/05-03-2018`
-* *OrderDelete* : `orderdelete INDEX` +
-e.g. `orderdelete 5`
-* *OrderList* : `orderlist`
-* *History* : `history`
-* *Theme* : `theme THEME_TYPE`
-* *OrderStatus* : `orderstatus INDEX os/ORDER STATUS`
-* *Undo* : `undo`
-* *Redo* : `redo`
-* *TopCustomers `coming in v2.0`* : `topcustomers NUMBER`
-* *TopItems `coming in v2.0`* : `topitems NUMBER`
-* *SendPromoGroup `coming in v2.0`* : `sendpromogroup GROUP... [sub/SUBJECT] [b/BODY]`
-* *SendPromoPref `coming in v2.0`* : `sendpromopref PREFERENCE... [sub/SUBJECT] [b/BODY]`
-* *Exit* : `exit`
-
-== Command Alias Summary
-
-* *Add* `a n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [g/GROUP]... [pr/PREFERENCE]...` +
-e.g. `a n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 g/friend g/colleague pr/computers`
-* *Calendar* : `cal [VIEW_FORMAT]` +
-e.g. `cal week`
-* *Calendar Next `coming in v1.5`* : `calnext`
-* *Calendar Previous `coming in v1.5`* : `calprev`
-* *Calendar Today `coming in v1.5`* : `calnow`
-* *Clear* : `c`
-* *Delete* : `d INDEX` +
-e.g. `d 3`
-* *GroupDelete* : `gd GROUP_NAME` +
-e.g. `gd friends`
-* *PreferenceDelete*: `pd PREFERENCE_NAME` +
-e.g. `pd computers`
-* *Edit* : `e INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [g/GROUP]... [pr/PREFERENCE]...` +
-e.g. `e 2 n/James Lee e/jameslee@example.com`
-* *EntryAdd* : `ea t/TITLE [sd/START_DATE] ed/END_DATE [st/START_TIME] et/END_TIME` +
-e.g. `ea t/roadshow sd/01-01-2019 ed/10-01-2019 st/0900 et/1800`
-* *EntryEdit `coming in v1.5`* : `ee e/ENTRY_INDEX [t/TITLE] [sd/START_DATE] [ed/END_DATE] [st/START_TIME] [et/END_TIME]` +
-e.g. `ee e/2 t/meeting with Jason ed/20-20-2020`
-* *EntryDelete* : `ed INDEX` +
-e.g. `ed 2`
-* *EntryList* : `el`
-* *EntryListClear `coming in v1.5`* : `elc`
-* *Find* : `f KEYWORD [MORE_KEYWORDS]` +
-e.g. `f James Jake`
-* *GroupFind* : `gf KEYWORD [MORE_KEYWORDS]` +
-e.g. `gf friends`
-* *PrefFind* : `pf KEYWORD [MORE_KEYWORDS]` +
-e.g. `pf computers`
-* *List* : `l`
-* *Help* : `?`
-* *Select* : `s INDEX` +
-e.g. `s 2`
-* *OrderAdd* : `oa INDEX i/ORDER INFORMATION pr/PRICE q/QUANTITY d/DELIVERY DATE` +
-e.g. `oa 1 i/NBA 2k18 pr/229.99 q/1 d/11-09-2018`
-* *OrderEdit* : `oe [i/ORDER INFORMATION] [pr/PRICE] [q/QUANTITY] [d/DELIVERY DATE]` +
-e.g. `oe 1 pr/15.00 d/05-03-2018`
-* *OrderDelete* : `od INDEX` +
-e.g. `od 5`
-* *OrderList* : `ol`
-* *History* : `h`
-* *Theme* : `t THEME_TYPE`
-* *OrderStatus* : `os INDEX os/ORDER STATUS`
-* *Undo* : `u`
-* *Redo* : `r`
-* *TopCustomers `coming in v2.0`* : `tc NUMBER`
-* *TopItems `coming in v2.0`* : `ti NUMBER`
-* *SendPromoGroup `coming in v2.0`* : `spg GROUP... [sub/SUBJECT] [b/BODY]`
-* *SendPromoPref `coming in v2.0`* : `spp PREFERENCE... [sub/SUBJECT] [b/BODY]`
-* *Exit* : `q`
+=== Managing Persons
```

Comment:
Not sure if its better to separate command word + Alias from the parameters? Would it look better? That way you can put the command word and alias column side by side, with parameter column to the right of alias?

Reply:
Ah, I understand. I'll update the tables.